src/tribol/interface/tribol.hpp

```diff
@@ -178,6 +179,62 @@ void registerMesh( integer meshId,
                    const real* y,
                    const real* z=nullptr );
 
+/*!
+ * @brief Define and register a coupling scheme over an MFEM mesh.
+ *
+ * This method is designed to enable simple registration of a contact coupling
+ * scheme over a parallel decomposed MFEM mesh.  Contact surfaces are defined
+ * via a list of boundary attributes, which is used to construct a contact
```

Comment:
Maybe elaborate a little more about the boundary attributes list e.g. "Each contact surface in the coupling scheme is defined via a list of one or more boundary attributes, which is used to construct a single contact surface mfem::ParSubMesh." Or, "The two contact surfaces in the coupling scheme are defined via lists of one or more boundary attributes per surface, which are used to construct surface mfem::ParSubMesh objects."

Reply:
Updated